src/main/scala/wdl4s/formatter/SyntaxFormatter.scala

```diff
@@ -98,6 +99,29 @@ class SyntaxFormatter(highlighter: SyntaxHighlighter = NullSyntaxHighlighter) {
     header
   }
 
+  private def formatRuntimeSection(task: Task, level: Int): String = {
+    task.runtimeAttributes.attrs match {
+      case m: Map[String, Seq[String]] if m.nonEmpty =>
```

Comment:
Sure, but my original question was why not just if/else on the nonEmpty. Although I suppose then you'd have to either pre-assign attrs or give the full reference twice. 

Either way I found the type ascription to be kinda confusing.